ui/js/search.js

```diff
@@ -101,13 +101,18 @@ require(['jquery', 'oae.core', 'underscore', 'jquery.history', 'jquery.switchtab
             $('#search-refine-type input[type="checkbox"][data-type="' + type + '"]').prop('checked', true);
         });
 
-        // Ensure only the selected tenant is chosen, if at all
-        $('#search-refine-tenant input[type="checkbox"]').removeAttr('checked');
-        if (queryData.tenant) {
-            $('#search-refine-tenant input[type="checkbox"][data-tenant="' + queryData.tenant + '"]').prop('checked', true);
+        // Hide the 'within tenant' checkbox for private tenants
+        var isPrivate = oae.api.config.getValue('oae-tenants', 'tenantprivacy', 'tenantprivate');
+        if (isPrivate) {
+            $('#search-refine-tenant').hide();
+        } else {
+            // Ensure only the selected tenant is chosen, if at all
```

Comment:
I'm not sure. I imagine it could've been set up this way for the case where a search is limited to a tenancy other than the user's own?